corona-warn-app/cwa-app-android
# [FEATURE] ENF delta (EXPOSURAPP-12210)

How to test:
- check database 'keyfiles' and storage data -> data -> de.coronawarnapp.test -> cache -> diagnosis-keys
- entries marked as checked should disappear on the next app start
- triggering key download should not result in download of files that are marked as checked
- only unmarked files are provided to ENF, marked after successful submission

currently targeting 2.19
